gnucash/CMakeLists.txt

```diff
@@ -45,7 +46,10 @@ set (gnucash_SOURCES
   )
 
 if (MINGW)
-  list(APPEND gnucash_SOURCES "gnucash-windows-locale.c")
+  list(APPEND gnucash_SOURCES "gnucash-locale-windows.c")
+endif()
```

Comment:
Should this be
```
if()
elseif()
endif()
```
Is there a scenario where both `MINGW` and `MAC_INTEGRATION` evaluate to true?

Reply:
Agreed using elseif() expresses the intent more clearly. I have pushed with your suggestion.